addons/event_sale/tests/test_event_specific.py

```diff
@@ -0,0 +1,59 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+import datetime
+
+from dateutil.relativedelta import relativedelta
+
+from odoo.addons.event_sale.tests.common import TestEventSaleCommon
+from odoo.tests.common import Form, tagged
+
+
+@tagged('post_install', '-at_install')
+class TestEventSpecific(TestEventSaleCommon):
+
+    def test_event_change_max_seat_no_side_effect(self):
+        """
+        Test that changing the Maximum (seats_max), the seats_reserved of all the ticket do not change
+        """
+        # Enable "sell tickets with sales orders" so that we have a price column on the tickets
+        config = self.env["res.config.settings"].create({
+            "module_event_sale": True
+        })
+        config.execute()
+        # Event template
+        with Form(self.env['event.type']) as event_type_form:
+            event_type_form.name = "Pastafarian Event Template"
+            event_type_form.use_ticket = True
+            # Edit the default line
+            with event_type_form.event_type_ticket_ids.edit(0) as ticket_line:
+                ticket_line.name = 'Pastafarian Registration'
+                ticket_line.price = 0
+            event_type = event_type_form.save()
+
+        with Form(self.env['event.event']) as event_event_form:
+            event_event_form.name = 'Annual Pastafarian Reunion (APR)'
+            event_event_form.date_begin = datetime.datetime.now() + relativedelta(days=2)
+            event_event_form.date_end = datetime.datetime.now() + relativedelta(days=3)
+            event_event_form.event_type_id = event_type  # Set the template
+            event_event_form.auto_confirm = True
+            # Create second ticket (VIP)
+            with event_event_form.event_ticket_ids.new() as ticket_line:
+                ticket_line.name = 'VIP (Very Important Pastafarian)'
+                ticket_line.price = 10
+            event_event = event_event_form.save()
+
+        # Add two registrations for the event
+        for t in event_event.event_ticket_ids:
```

Comment:
ticket ?

Reply:
I can't infer a lot of information of that one single word, but I think that for loop is correct, I'm basically creating 1 registration for each ticket, I thought it would be clearer, I can unwrap the loop and do
`self.env['event.registration'].create({
                'event_id': event_event.id,
                'event_ticket_id': event_event.event_ticket_ids[0].id
            })
self.env['event.registration'].create({
                'event_id': event_event.id,
                'event_ticket_id': event_event.event_ticket_ids[1].id
            })
`
if you want, or update the comment

Reply:
Was just proposing to rename t to ticket :D Even for loops I prefer meaningful variable names.

Reply:
Alrighty